api/models.py

```diff
@@ -1,16 +1,43 @@
 # api/models.py
 from django.db import models
-from django.contrib.auth.models import User
+#from django.contrib.auth.models import User
```

Comment:
PR should contain production-ready code. If you want to remove something, do not comment it - just remove it